package.json

```diff
@@ -50,87 +52,84 @@
   "author": "codeBelt",
   "license": "MIT",
   "devDependencies": {
-    "@types/classnames": "2.2.3",
-    "@types/fs-extra": "5.0.2",
-    "@types/hapi": "17.0.10",
-    "@types/history": "4.6.2",
-    "@types/inert": "5.1.0",
-    "@types/node": "9.6.5",
+    "@babel/cli": "7.1.5",
+    "@babel/core": "7.1.5",
+    "@babel/plugin-proposal-class-properties": "7.1.0",
+    "@babel/plugin-proposal-object-rest-spread": "7.0.0",
```

Comment:
do you still need this?  even if babel needs it, isn't this covered by `tsc`?

Reply:
Not 100% but I am using Babel 7 to compile the `ts` now and not `tsc`

Reply:
Let's talk about that tomorrow.  Why would you use an abstraction of ts to compile ts?